src/service/state.js

```diff
@@ -155,6 +155,27 @@ module.exports = (statePersistence, municipalityService, seService) => {
       return { total_area: stateArea[0].area };
     },
 
+    /**
+     * Get the information the current area distribution for each human footprint category in the
+     * given state
+     * @param {Number} stateId state id
+     *
+     * @returns {Object[]} Array of areas by human footprint category with their respective
+     * percentage
+     */
+    getAreaByHFCategory: async (stateId) => {
+      let stateArea = await state.getTotalArea(stateId);
+      stateArea = stateArea.total_area;
+      console.log('stateArea', stateArea);
```

Comment:
Remove console.log, I'll try to configure the action to report an error instead of a warning.

Reply:
Done